data/json/recipes/tools/tool.json

```diff
@@ -492,8 +492,15 @@
     "time": "200 m",
     "reversible": true,
     "book_learn": [ [ "textbook_chemistry", 4 ], [ "brewing_cookbook", 3 ] ],
-    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 2 } ],
-    "components": [ [ [ "pot", 1 ], [ "pot_copper", 1 ] ], [ [ "hose", 2 ], [ "makeshift_hose", 2 ] ] ]
+    "proficiencies": [ { "proficiency": "prof_metalworking" } ],
+    "tools": [ [ [ "material_sand", 1 ] ] ],
```

Comment:
Treating it as a tool enables not (incorrectly) using up the sand?

Reply:
I don't believe the sand saw supposed to be "used up". As far as I understand the idea you just keep it in the copper tube when bending the tube to prevent it from collapsing.

Reply:
Correct. In the initial issue it was proposed to be used as both a component and a byproduct in order to model getting it back afterward.

Reply:
Using it a tool just felt more natural to me.